content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Operations Start Guide
+type: docs
+---
+
+## Introduction
+
+The Operations Start Guide will help you start with EGI Operations duties. It
+covers the responsibilities of the various parties involved in the running of
+the [EGI infrastructure](https://ims.egi.eu/display/EGIG/EGI+Infrastructure) and
+guide how to join operations. As a newcomer, you need to understand the
+structure of the infrastructure and roles of operators at different levels.
+Reading the whole document will give you a complete overall picture of daily
+operations within EGI.
+
+## Resource Centres and Resource Infrastructures
+
+Resources are geographically distributed and are contributed by
+[Resource Centres](https://ims.egi.eu/display/EGIG/Resource+Centre). A Resource
+Centre is the smallest resource administration domain within EGI. It can be
+either localized or geographically distributed. A Resource Centre is also known
+as a _site_. It provides a minimum set of local or remote IT Services, such as
+High Throughput Compute (HTC), Cloud Compute, Storage and Data Transfer,
+compliant to well-defined IT Capabilities necessary to make resources accessible
+to authorised users. Users can access the services using common interfaces.
+
+The EGI Federation is a
+[Resource Infrastructure](https://ims.egi.eu/display/EGIG/Resource+Infrastructure)
+federating Resource Centres to constitute a homogeneous operational domain, and
+the
+[Resource Infrastructure Provider](https://ims.egi.eu/display/EGIG/Resource+Infrastructure+Provider)
```

Comment:
@paolini78 here we mention Resource Infrastructure Providers, while in [PROC02](https://confluence.egi.eu/display/EGIPP/PROC02+Operations+Centre+creation) we are mentioning Operations Centre, my understanding is that it's the same thing, so it may be a good opportunity to align all this. See also https://jira.egi.eu/browse/IMSIS-208.

Reply:
Ping @mviljoen-egi.

Reply:
they are not the same: https://ims.egi.eu/display/EGIG/Operations+Centre

Reply:
So you mean it's the same as a Resource Center? or is it yet again a different entity?
The fact is that Operations Center is not mentioned in the intro of this guide.

Reply:
I'm quite confused (see https://jira.egi.eu/browse/IMSIS-208), initially I thought that Operations Center were basically NGIs which were also named at some places Resource Infrastructure Providers, but I fail to understand the subtleties, especially as it's doesn't seem to be used everywhere in the same way. We have PROC09 for resource centre registration and certification, PROC11 for Resource Center decommissioning, but nothing about Resource Infrastructure Providers, only something about Operations Center (PROC02 and PROC03), which is confusing.

Reply:
:DDDD
NGIs are just an example of Resource Infrastructure Providers and the OC it is just the collection of supporting services (human and technical) of that specific NGIor RIP in general. I'll try to add a sentence somewhere...

Reply:
But what is the benefit of having the term Operations Center used? Wouldn't it be easier to just have PROC02 and PRO03 renamed to cover RIP? What is an RIP beyond it's supporting services (human and technical)? There isn't a one to one mapping between a RIP and and an operations centre?

Reply:
at the end the OC is part of a RIP, like the RCs; we usually interface with the OCs rather than with the RIP.
Let's see if we can improve the wording.